test/tree.spec.ts

```diff
@@ -1086,4 +1086,32 @@ describe('Tree', () => {
     expect(masterTree.children[1].leftMenuTemplate).toEqual('<i class="navigation"></i>');
   });
 
+  it('should load children when hasChildren is true', () => {
+
+    const model: TreeModel = {
+      value: 'root',
+      hasChildren: true,
```

Comment:
@TMaster hasChildren property means that we tree has children, but under the hood, it means: "Notify me when I can load more children", right? Maybe we can name it a bit more precisely? What do you think?

Reply:
Exactly, It suppose to mean there is mean `There are more children out there in the back end, just let me know and I will load them`. More suitable name perhaps may be something like:
- deferredChildrenExists. 
- additionalChildrenExists.
- additionalLoadingAvailable.

@rychkog what do you say?

Reply:
Something like (taking advanatge over LoadNextLevelEvent name)
`emitLoadNextLevelEvent: true`
Smth like that. Or maybe `notifyAboutNextLevelChildrenLoading: true`. Just to make it self explanatory - final decision is up to you